README.md

```diff
@@ -222,7 +222,11 @@ RabbitMQ Environment Variables in rabbitmq_env.config
 The erlang cookie to use for clustering - must be the same between all nodes.
 This value has no default and must be set explicitly if using clustering.
 
-###`key_content`
+####`file_limit`
+
+Set rabbitmq file ulimit. Defaults to 16384. Only available on systems with `$::osfamily == 'Debian'`
```

Comment:
Because that's what was implemented? [This](https://github.com/puppetlabs/puppetlabs-rabbitmq/blob/15e2421048a3495e60482837910441a1248bf6f0/manifests/config.pp#L101-L110) is the only place `file_limit` is used. I didn't know enough about rabbitmq on other oses to change that, I just documented and tested what i found.